index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="resources/css/style.css">
+    <title>HTML SIGN UP FORM</title>
+</head>
+
+<body>
+    <nav class="main">
+        <label class="inner-nav">
+            <p>Already have an Intuit account?
+                    <a href="#"> I forgot my User ID or Password </a></p>
+            <button>
+                <img src="./resources/img/lock-icon.png" alt=" lock icon"> Sign In</button>
+        </label>
+    </nav>
+
+    <header class="header">
+        <nav class="logo">
+            <img src="resources/img/download.png" alt="Intuit logo">
+        </nav>
+        <ul>
+            <li><img src="./resources/img/mint.png" alt="Mint logo"> Mint</li>
+            <li><img src="./resources/img/quickbook.png" alt="QuickBooks logo"> Quickbooks</li>
+            <li><img src="./resources/img/turbo.jpeg" alt="Turbotax logo"> Turbotax</li>
+        </ul>
+    </header>
+
+    <section>
+        <div class="form">
+            <h1>Create an Intuit account</h1>
+        <h2>One account for everything Intuit, including Mint.</h2>
+        <a class="learn-more" href="#">Learn more</a>
+        <form action="/" method="POST">
+            <label>
+                Email Address
+                <input type="email" name="email">
+            </label>
+            <label>
+                Phone (recommended)
+                <input type="tel" name="email">
+                <span>Standard call, messaging or data rates may apply.</span>
+            </label>
+            <label>
+                Password
+                <input type="password">
+            </label>
+            <button class="create-account-btn"> <img src="./resources/img/lock-icon.png" alt=" lock icon">Create an account</button>
```

Comment:
Please use an `<input>` type submit in here and style it as a button (every form needs a submit input a simple button tag won't work in this case)